src/Sarif/Writers/SarifLogger.cs

```diff
@@ -149,11 +149,9 @@ private static string Redact(string text, IEnumerable<string> tokensToRedact)
 
         private void SetSarifLoggerVersion(Tool tool)
         {
-            tool.Properties = tool.Properties ?? new Dictionary<string, string>();
```

Comment:
nice

---
In reply to: [62750560](https://github.com/Microsoft/sarif-sdk/pull/257#discussion_r62750560) [](ancestors = 62750560)